src/coreclr/ToolBox/superpmi/superpmi/commandline.cpp

```diff
@@ -100,6 +100,10 @@ void CommandLine::DumpHelp(const char* program)
     printf("     If 'workerCount' is not specified, the number of workers used is\n");
     printf("     the number of processors on the machine.\n");
     printf("\n");
+    printf(" -failureLimit [limit]\n");
```

Comment:
The brackets imply that `limit` is optional. I would remove them, or replace them with `<` and `>` so:
```
printf(" -failureLimit <limit>\n");
```